x-pack/solutions/observability/plugins/slo/public/pages/slos/components/badges/slo_badges.tsx

```diff
@@ -42,6 +42,7 @@ export function SloBadges({
       ) : (
         <>
           <SloStatusBadge slo={slo} />
+          <SloStateBadge slo={slo} />
```

Comment:
IMO, this is adding bit more noise on the card view or other places, possibly we can omit the inidactor type badge

Reply:
You're suggesting to remove the indicator type badge from there?